README.md

```diff
@@ -1,4 +1,262 @@
 IPFS RFCs
 =====
 
-This repository will be the place where people propose, comment on, and refer to RFCs for changes to IPFS and its surrounding projects. Its initial manifestation will probably look a lot like the [Rust lang RFCs repository](https://github.com/rust-lang/rfcs)
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ipfs-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+IPFS, libp2p or the RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Any semantic or syntactic change to the protocols that is not a bugfix.
+  - Removing protocol features, including those that are feature-gated.
+  - Adding anything to our list of officially supported projects, libraries, and specifications.
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap more errors, etc.)
+  - Additions only likely to be _noticed by_ other developers-of-ipfs, but
+    invisible to users-of-ipfs.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+
+### working group specific guidelines
+[working group specific guidelines]: #working group-specific-guidelines
```

Comment:
This is rendering in plain text — I think because there’s a space instead of a dash in `#working group`.

Reply:
Or is this actually supposed to be a reference to Rust, since you’re saying that’s what this is similar to? In either case, the link is wrong :P